license-classifications.yml

```diff
@@ -1431,17 +1431,12 @@ categorizations:
     categories:
       - "commercial"
 
+  # https://spdx.org/licenses/Classpath-exception-2.0.html
   - id: "Classpath-exception-2.0"
     categories:
       - "property:AutoConf-or-other-exception"
       - "property:include-in-notice-file"
 
-  # Duplicate of the previous classification due to tooling requirements.
-  - id: "LicenseRef-Classpath-exception-2.0"
```

Comment:
Unrelated to this change per se, but actually stand-alone exceptions (to licenses) should never be classified on their own at all, IMO.

Reply:
I'll probably got to those in a later batch. Not sure if this specific one is included, but I will keep an eye out for sure.